clients/src/main/resources/common/message/DescribeQuorumResponse.json

```diff
@@ -40,10 +45,27 @@
         { "name": "CurrentVoters", "type": "[]ReplicaState", "versions": "0+" },
         { "name": "Observers", "type": "[]ReplicaState", "versions": "0+" }
       ]}
-    ]}],
+    ]},
+    { "name": "Nodes", "type": "[]Node", "versions": "2+", "fields": [
+      { "name": "NodeId", "type": "int32", "versions": "2+",
+        "mapKey": true, "entityType": "brokerId", "about": "The ID of the associated node" },
+      { "name": "Listeners", "type": "[]Listener",
+        "about": "The listeners of this controller", "versions": "2+", "fields": [
+        { "name": "Name", "type": "string", "versions": "2+", "mapKey": true,
+          "about": "The name of the endpoint" },
+        { "name": "Host", "type": "string", "versions": "2+",
+          "about": "The hostname" },
+        { "name": "Port", "type": "uint16", "versions": "2+",
+          "about": "The port" },
+        { "name": "SecurityProtocol", "type": "int16", "versions": "2+",
```

Comment:
I don't see why we need `SecurityProtocol` here. We haven't exposed it before and it can't be set by `AddRaftVoter`, etc. cc @jsancio

Reply:
Also if we have it, we need some way of handling new security protocols in the future (SecurityProtocol.UNKNOWN, etc.)